jobs/adspert_python_dev.html

```diff
@@ -0,0 +1,48 @@
+---
+title: Python Software Developer
+company: Adspert (Bidmanagement GmbH)
+url: https://www.adspert.net/en/jobs/python-software-developer-mf/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Karl Bartel
+    email: jobs@adspert.net
+created: !!timestamp '2016-05-08'
+tags:
+  - python
+  - sql
+---
+
+**Note: The position is full-time and based in our Berlin office, therefore if you are outside of Berlin you must be prepared to relocate for this role.**
+
+Adspert was founded in Berlin in 2010 and our innovative technology launched just one year later. Based on trading systems, Adspert automatically organizes and optimizes online advertising. Bertelsmann Digital Media Investment and the Bell Group's support Adspert as strategic partners and investors.
+
+Today, around 20 Adspert employees are hard at work in the heart of Berlin, passionately implementing the vision of developing bid management of the future.
+
+**We offer:**
+
+We develop our solutions with Python, Go, JavaScript and PostgreSQL and mainly work on Linux and Mac. Distributed version control, test-driven development and automated deployment processes are basic working methods. With us, you can expect a friendly atmosphere and a healthy attitude to the balance between work and leisure.
+
+**Your profile:**
+
+*   Bachelor’s in computer science or a discipline with computer science share or equivalent practical experience.
+*   Very good English language skills, German language skills are a bonus.
+*   Advanced Python skills, have already been developed in JavaScript and ideally also made first experiences with Go.
```

Comment:
This is grammatically incorrect. I suggest this alternative wording: "The candidate should have advanced Python skills and be familiar with JavaScript and Go".